Author: Corey Bryant <corey.bryant@canonical.com>
Last-Update: 2016-05-18
Forwarded: not-needed

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -140,7 +140,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
 
 # Add any extra paths that contain custom files (such as robots.txt or
 # .htaccess) here, relative to this directory. These files are copied
